etc/build/push.sh

```diff
@@ -49,7 +49,7 @@ if ls CHANGELOG-*.md; then
   ) > CHANGELOG.md.new
   mv CHANGELOG.md.new context/app/markdown/CHANGELOG.md
   git rm CHANGELOG-*.md
-  git add .
+  git add context/app/markdown/CHANGELOG.md
```

Comment:
We also have to commit the file deletions at the project root?

Reply:
`git rm` stages the removals; I just tested this locally to be sure:

```
(portal) nikolay@NikolayPCWin:~/projects/hubmap/portal-ui$ git rm example-app.conf 
rm 'example-app.conf'
(portal) nikolay@NikolayPCWin:~/projects/hubmap/portal-ui$ git status
On branch nickakhmetov/hmp-348/misc-dependency-updates
Your branch is up to date with 'origin/nickakhmetov/hmp-348/misc-dependency-updates'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        deleted:    example-app.conf
```